etc/profile-a-l/balsa.profile

```diff
@@ -0,0 +1,67 @@
+# Firejail profile for balsa
+# Description: GNOME mail client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include balsa.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.balsa
+# Comment to use GPG
+noblacklist ${HOME}/.gnupg
+noblacklist ${HOME}/mail
+noblacklist /var/spool/mail
```

Comment:
- `noblacklist /var/mail`
- no `whitelist`?

Reply:
I mean you need /var/mail and /var/spool/mail, compare mutt or thunderbird. And if you `include whitelist-var-common.inc` you need to `whitelist` these locations under /var.

Reply:
I missed this comment, I was going by balsa's config box which needs /var/mail..